libs/category/src/actions/category.actions.ts

```diff
@@ -27,7 +31,34 @@ export class DaffCategoryLoadFailure implements Action {
   constructor(public payload: string) { }
 }
 
+export class DaffChangeCategoryPageSize implements Action {
+  readonly type = DaffCategoryActionTypes.ChangeCategoryPageSizeAction;
+
+  constructor(public payload: number) { }
```

Comment:
What are these payloads? Either they need documentation, or need proper naming, or more realistically, both.

Reply:
Added docs and changed the payload names